packages/connect/src/device/Device.ts

```diff
@@ -529,15 +517,28 @@ export class Device extends TypedEmitter<DeviceEvents> {
             feat.internal_model = ensureInternalModelFeature(feat.model);
         }
 
-        this.features = feat;
-        this.featuresNeedsReload = false;
-
         this.firmwareType =
             feat.capabilities &&
             feat.capabilities.length > 0 &&
             !feat.capabilities.includes('Capability_Bitcoin_like')
                 ? 'bitcoin-only'
                 : 'regular';
+
+        const version = [feat.major_version, feat.minor_version, feat.patch_version];
+        // capabilities could change in case where features was fetched with older version of messages.json which doesn't know this field
```

Comment:
sidenote: this comment does not apply anymore. we used to have multiple versions of messages but this has been removed long time ago.

Reply:
so we can remove the comment or also the capabilitiesDidChange check? I guess that even the `capabilitiesDidChange` const can be removed?

Reply:
yes, I'd say it can be removed. or at least I hope that 
```
versionCompare(version, this.getVersion()) !== 0
```
does the job well enough. 

also no need to do it inside this PR. its up to you.